MILESTONES.md

```diff
@@ -0,0 +1,152 @@
+# Milestones
+
+(What's next for the prototype?)
+
+## M0.1:
+
+### Goals:
+
+- JetBrains will write out the contents of the active editor to a temporary
+  file (or just a file that's namespaced under the JetBrains state folder) for
+  VS Code to view.
+    - VS Code will _not_ open the actual file, because we don't want to have to
```

Comment:
We may want to consider opening the actual file, but then using the contents sent to us by JetBrains.  I don't think that's necessary today, but help with things like detecting programming language, potential future actions that refer to file name, etc

Reply:
I was wondering about that. We could always make the temporary file match the same file extension, but I was curious if Cursorless plans to do any analysis at the project level, so it would include other files beyond the current one (e.g. indexing symbols). In which case just mirroring the single file would miss out on that.

Reply:
For cross-file semantic analysis, Cursorless today relies on the IDE. For example for the "rename" action, we just select the token and trigger the VSCode rename command. Same with "follow" (jump to definition)